build.gradle

```diff
@@ -13,7 +13,7 @@ plugins {
 apply plugin: 'project-report'
 
 description = 'JGiven - BDD in plain Java'
-def jacocoEnabled = !JavaVersion.current().isJava11Compatible()
+def jacocoEnabled = !JavaVersion.current().isJava12Compatible()
```

Comment:
Why do we change from 11 to 12 at this place?

Reply:
Simple, we don't need Jacoco to run for all versions. Previously, it ran only for Java8, so due to bumping up to 11 we now only run it for Java 11.

Reply:
Sorry, so you meant we only need to run it for Java 12?

Reply:
Not exactly, we run it for Java 11 and omit it for all versions above that

Reply:
:facepalm:  Yeah, I see. Thanks for the clarification!